index.html

```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
                 </li>
             </ul>
         </section>
+
+        <section class="works wrapper">
+            <h2 class="sr-only">Works Section</h2>
+            <article class="card">
+                <img src="./images/Tonic.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Tonic</h3>
```

Comment:
[OPTIONAL] Kindly add the `alt` attribute, it's very useful if the image didn't load. it's a good practice.